### PR TITLE
fix table view for exponential notation examples

### PR DESCRIPTION
**Pull request by andreasantillana**
I think the table's format was in markdown, so I switched it to adoc.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -4319,11 +4319,12 @@ When using exponential notation for numbers, prefer using the normalized scienti
...
-|----------------|----------------------|----------|
-| 10 ** 7        | 1e7                  | 10000000 |
-| 10 ** 6        | 1e6                  | 1000000  |
-| 10 ** 7        | 10e6                 | 10000000 |
+|===
+| power notation | exponential notation | output
+| 10 ** 7        | 1e7                  | 10000000
+| 10 ** 6        | 1e6                  | 1000000
+| 10 ** 7        | 10e6                 | 10000000
+|===
```
Comment (koic) [inline README.adoc:4327]: Good catch! Can you insert a blank line after the header?

```suggestion
|===
| power notation | exponential notation | output

| 10 ** 7        | 1e7                  | 10000000
| 10 ** 6        | 1e6                  | 1000000
| 10 ** 7        | 10e6                 | 10000000
|===
```

**Review by koic — APPROVED** [review]
This looks good! Can you squash your commits into one?

**Comment by bbatsov** [comment]
I squashed them myself. Thanks for fixing this!

**Comment by andreasantillana** [comment]
Thanks @bbatsov!

**Pull request merged**